src/apps/backend/modules/otp/otp-service.ts

```diff
@@ -0,0 +1,37 @@
+import config from 'config';
+
+import { PhoneNumber } from '../account/types';
+import { SMSService } from '../communication';
+
+import OtpWriter from './internal/otp-writer';
+import {
+  Otp,
+} from './types';
+
+export default class OtpService {
+  public static async createOtp(
+    phoneNumber: PhoneNumber,
+  ): Promise<Otp> {
+    const otp = await OtpWriter.expirePreviousOtpAndCreateNewOtp(phoneNumber);
+
+    const nodeConfigEnv = config.get('nodeConfigEnv');
+    const isProductionEnv = nodeConfigEnv === 'production';
+    const isTestingEnv = nodeConfigEnv === 'testing';
+
+    if (isProductionEnv || isTestingEnv) {
```

Comment:
I don't think this code should live here, but rather should live in SMS service.

A better way to do is write feature specific config flag, so insteaed of check env, add a flag - enableSMS flag in config files and have SMS service check that and do what it needs to do.